tools/opt/opt.cpp

```diff
@@ -155,6 +155,9 @@ NOTE: The optimizer is a work in progress.
   --merge-return
                Replace all return instructions with unconditional branches to
                a new basic block containing an unified return.
+
+               This pass does not currently support structured control flow. It
+               bails out if the shader capability is detected.
```

Comment:
"Bail out" is colloquial and ambiguous: does it result with success or failure?

Suggest replacing "bails out" with "does nothing" 
(Also in the method comments talking about bailing out).